app/queries/comments/community_wellness_query.rb

```diff
@@ -65,5 +85,58 @@ def self.sql_query
         GROUP BY user_id
       SQL
     end
+
+    # This query is an exact copy of the `sql_query` but includes a filter that
+    # forces the weeks to start counting from May 1st, 2022. This allows us to
+    # rollout this badge one week at a time without human intervention.
+    #
+    # We can officially remove this query+logic on December 19, 2022 at which
+    # time we will continue to use the default query.
+    def self.limit_release_date_sql_query
+      <<~SQL
+        SELECT user_id,
+               /* A comma separated string of "weeks_ago" */
+               array_to_string(array_agg(weeks_ago), ',') AS serialized_weeks_ago,
+               /* A comma separated string of comment counts. The first value in this string happens on the week that is the first value in serialized_weeks_ago */
+               array_to_string(array_agg(number_of_comments_with_positive_reaction), ',') AS serialized_comment_counts
+        FROM
+            (
+                SELECT user_id,
+                       COUNT(user_id) AS number_of_comments_with_positive_reaction,
+                       /* Get the number of weeks, since today for posts */
+                       trunc((extract(epoch FROM (current_timestamp- created_at))) / 604800) AS weeks_ago
+                FROM comments
+                INNER JOIN
+                    (
+                          SELECT DISTINCT reactable_id
+                          FROM reactions
+                          WHERE reactable_type = 'Comment'
+                          AND created_at > (now() - interval '231' day)
+                          EXCEPT
+                          SELECT DISTINCT reactable_id
+                          FROM reactions
+                          WHERE reactable_type = 'Comment'
+                          AND created_at > (now() - interval '231' day)
+                          AND category IN ('thumbsdown', 'vomit')) AS negative_reactions
+                ON comments.id = negative_reactions.reactable_id
+                INNER JOIN
+                    (
+                          SELECT count(id) AS number_of_comments,
+                                 user_id AS comment_counts_user_id
+                          FROM comments
+                          /* This interval filters week 1 (what we care about) */
+                          WHERE created_at >= (now() - interval '14' day)
+                          AND created_at <= (now() - interval '7' day)
+                          GROUP BY user_id) AS comment_counts
+                          ON comments.user_id = comment_counts_user_id
+                          AND comment_counts.number_of_comments > 1
+                          /* Don’t select anything older than 231 days ago, or 33 weeks ago */
+                          WHERE created_at > (now() - interval '231' day)
+                          /* We will only awarded from this date forward (feature release date) */
+                          AND created_at > '2022-05-01'
```

Comment:
This SQL query is identical to the original one with this exception. The Blazer query ([available here](https://dev.to/admin/blazer/queries/683)) returns at most 2 weeks (week 0 and week 1). The Explain Analyze of this ([available here](https://dev.to/admin/blazer/queries/691)) appears nearly identical to the unrestricted version (checks the same indexes as the original one)